internal/handlers/common/serverstatus.go

```diff
@@ -59,6 +64,15 @@ func ServerStatus(startTime time.Time, cm *connmetrics.ConnMetrics) (*types.Docu
 		}
 	}
 
+	telemetryState := "disabled"
+
+	switch {
+	case state.Telemetry == nil:
+		telemetryState = "undecided"
+	case pointer.GetBool(state.Telemetry):
+		telemetryState = "enabled"
+	}
```

Comment:
Don't we want to define `disabled`, `undecided`, `enabled` as constants in the telemetry related package? It seems that those words are met a few times in the source code.

Reply:
https://github.com/FerretDB/FerretDB/pull/1380/commits/4d18588abe0fd2bf50932edc0c4d9f7cb755dd54 something like that?

Reply:
@noisersup yeah, looks good to me! (Linter complains as the comment should be either out of the `const` block or there should be dedicated comments for each cont).